addons/pause/userscript.js

```diff
@@ -16,6 +16,16 @@ export default async function ({ addon, global, console, msg }) {
   onPauseChanged(setSrc);
 
   while (true) {
+    if (!document.querySelector(".pause-btn")) {
+      document.addEventListener("keydown", function (e) {
+        if (addon.settings.get("keybindEnabled")) {
+          if (e.altKey && e.key === "≈") {
```

Comment:
What is this key?

Reply:
> What is this key?

It's the key for x when alt is pressed at the same time.

Reply:
On which keyboard layout?

Reply:
Macbook Air, just normal keyboard. At first when I tried checking if the key was "x", it wouldn't work. When I checked what the key was, it had "≈".

Reply:
That's probably specific to Mac keyboards. Try using `e.code` instead.